manual/building.rst

```diff
@@ -60,7 +60,7 @@ which are specified in the file ``Color Profile EULA.pdf``.
 Mac OS X
 ========
 
-You will need Homebrew_. Then:
+You will need Homebrew_ and at least OSX 10.11.5. Then:
```

Comment:
"change what you're saying, don't change what you've said" (Eels)

OSX is the official name for versions before 10.12, it was the name then:
https://web.archive.org/web/20160902012446/https://www.apple.com/osx/

And still is is according to Apple (text and pictures):
https://www.apple.com/macos/how-to-upgrade/

But `macOS` is the new mantra, so I guess we should change the title ;)